docs/src/further_topics/controlling_merge.rst

```diff
@@ -0,0 +1,198 @@
+        Dimension coordinates:
+            model_level_number                             x            -               -
+            latitude                                       -            x               -
+            longitude                                      -            -               x
+        Auxiliary coordinates:
+            level_height                                   x            -               -
+            sigma                                          x            -               -
+            surface_altitude                               -            x               x
+        Derived coordinates:
+            altitude                                       x            x               x
+        Scalar coordinates:
+            forecast_period             1339560.0 hours, bound=(1339200.0, 1339920.0) hours
+            forecast_reference_time     2006-01-01 00:00:00
+            time                        2161-01-16 00:00:00, bound=(2161-01-01 00:00:00, 2161-02-01 00:00:00)
+        Cell methods:
+            0                           time: mean (interval: 1 hour)
+        Attributes:
+            STASH                       m01s00i002
+            source                      'Data from Met Office Unified Model'
+            um_version                  '12.1'
+
+These two cubes vary along the time dimension, however, we are not able to combine them with :meth:`~iris.cube.Cube.merge`
+due to the fact that their ``surface_altitude`` coordinate also varies over time.
+
+    >>> cubes.merge_cube()
+    Traceback (most recent call last):
+    ...
+    iris.exceptions.MergeError: failed to merge into a single cube.
+      Coordinates in cube.aux_coords (non-scalar) differ: surface_altitude.
+
+Since surface altitude is preventing merging, we want to find a way of combining these cubes while also *explicitly*
+combining the ``surface_altitude`` coordinate so that it also varies along the time dimension. We can do this by first
+adding a dimension to the cube *and* the ``surface_altitude`` coordinate using :func:`~iris.util.new_axis`, and then
+concatenating those cubes together. We can attempt this as follows:
+
+    >>> from iris.util import new_axis
+    >>> from iris.cube import CubeList
+    >>> test_cubes = CubeList([new_axis(cube, scalar_coord="time", expand_extras=["surface_altitude"]) for cube in cubes])
+    >>> test_cubes.concatenate_cube()
+    Traceback (most recent call last):
+    ...
+    iris.exceptions.ConcatenateError: failed to concatenate into a single cube.
+      Scalar coordinates values or metadata differ: forecast_period != forecast_period
+
+This error alerts to us that the ``forecast_period`` coordinate is also varying over time. To get concatenation to work,
+we will have to expand the dimension of this coordinate also by passing it to the ``expand_extras`` keyword.
+
+    >>> processed_cubes = CubeList(
```

Comment:
As this is a "retry" of the previous failed command, I'd suggest that the result should have the same name - i.e. both should be either "test_cubes" or "processed_cubes".